mkosi/machine.py

```diff
@@ -129,9 +133,9 @@ def boot(self) -> None:
             # We use pexpect to boot an image that we will be able to interact with in the future.
             # Then we tell the process to look for the # sign, which indicates the CLI for that image is active.
             # Once we've build/boot an image the CLI will prompt "root@image ~]# ".
-            # Then, when pexpects finds the '#' it means we're ready to interact with the process.
+            # Then, when pexpects finds the ']#' it means we're ready to interact with the process.
             self._serial = pexpect.spawnu(cmd, logfile=None, timeout=240)
-            self._serial.expect("#")
+            self._serial.expect("]#")
```

Comment:
This is the shell I get when I boot an opensuse image in qemu:

```
localhost:~ # 
```

Reply:
Yeah, I just wanted to see if it'd pass the current integration tests in the CI.